src/OrchardCore.Modules/OrchardCore.Cors/Assets/Admin/cors-admin.js

```diff
@@ -70,24 +70,26 @@ var corsApp = new Vue({
             if (policy.isDefaultPolicy) {
                 this.policies.forEach(p => p.isDefaultPolicy = false);
             }
-            if (policy.OriginalName) {
+
+            if (policy.originalName) {
                 var policyIndex = this.policies.findIndex((oldPolicy) => oldPolicy.name === policy.originalName);
                 this.policies[policyIndex] = policy;
             }
             else {
                 this.policies.push(policy);
             }
+
             this.save();
             this.back();
         },
         save: function () {
-            document.getElementById('CorsSettings').value = JSON.stringify(this.policies);
+            document.getElementById('corsSettings').value = JSON.stringify(this.policies);
             document.getElementById('corsForm').submit();
         },
         back: function () {
             this.selectedPolicy = null;
         },
-        searchBox: function() {
+        searchBox: function () {
             var searchBox = $('#search-box');
```

Comment:
Here we use a - separator. Maybe consistency would require that we always use this separator for id attributes.

Reply:
> Also formatting of this file comes from where?

VS

Reply:
Thanks, was just for info.